plugins/rollbar-backend/src/api/RollbarApi.ts

```diff
@@ -103,6 +103,44 @@ export class RollbarApi {
     );
   }
 
+  async updateItem(
+    projectId: number,
+    itemId: number,
+    body: { assigned_user_id: string; }
+  ) {
+    const accessToken = await this.getAccessTokenByProjectId(projectId);
+    return this.patch(`/item/${itemId}`,
+      body,
+      accessToken,
+    )
+  }
+
+  private async getAccessTokenByProjectId(projectId: number) {
```

Comment:
I used this because I don't have the projectName, just the id

Reply:
I'm confused, is that not what the existing method needs? 😅